release.sh

```diff
@@ -2,32 +2,63 @@
 
 set -e
 
-last_tag=$(git tag --sort=-creatordate | head -n 1)
```

Comment:
Is there any significance in this change? I doubt this was required or our versioning would sometime screw up. cc @mesaugat

Reply:
I was testing and I forget to erase it, well looked.